src/app.js

```diff
@@ -30,6 +30,10 @@ export const canvas = new window.p5(sketch => { // eslint-disable-line
   const startBalls = () => {
     let id = 0
     const peopleWithAppInPercentage = document.getElementById('app_users')
+    STARTING_BALLS[STATES.infected] = document.getElementById('infected_people').value
+    const maxMovementSpeed = (document.getElementById('max_movement_speed').value * 0.01)
```

Comment:
Just wondering why the 0.01 factor here instead of smaller values in the Select's options. Maybe to transform implicitly to a number? Anyway, this is not a problem right now so this is just a comment

Reply:
It's cause 100% is the regular velocity. I believe it's easier to have a multiplier here than to map the select options to different values

Reply:
Replace with a constant